docker/Dockerfile.binary

```diff
@@ -0,0 +1,21 @@
+#
+# Dockerfile that builds the SurrealDB Linux binary and makes it depend on GLIBC 2.17.
+#
+
+FROM docker.io/ubuntu:18.04
```

Comment:
Just curious if you had any problems using a distroless image here? I think they support glibc.

Reply:
I use this Dockerfile only for building the binary, and the binary is then extracted out of the image to generate the release artifacts.

With a distroless image can you build binaries? I see the value when you actually want to run the resulting image (we chose chainguard images for that), but I use this Dockerfile as an alternative for building the binary on the github runner itself.